app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,25 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Food.all
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = Food.new(food_params)
+    @food.user_id = 1
```

Comment:
Here it would be better to set the `user_id` of the food to the current_user id. Setting it to one would be associating foods created by different users to the `user` with `id` of one.